Simple solvers/DO.c

```diff
@@ -11,32 +12,34 @@ int main() {
     scanf("%lf", &c);
     printf("\n");
     discriminant = (b * b - 4 * a * c);
-//Quadratic eqation way    
+
+//Quadratic variables  
     double root1, root2;
-    
+
+//Using the Quadratic equation way
     if (discriminant > 0) {
         root1 = (-b + sqrt(discriminant)) / (2 * a);
         root2 = (-b - sqrt(discriminant)) / (2 * a);
         ("root1 = %lf", root1);
         ("root2 = %lf", root2);
     }
-//Print roots from Quadratic
+//Print roots from Quadratic equation

```

Comment:
Typically we put a space between the comment token and the comment itself.  Consider doing this for all your comments.
```suggestion
// Print roots from Quadratic equation

```

Reply:
Please do this for ALL comments